userbenchmark/functorch/__init__.py

```diff
@@ -0,0 +1,31 @@
+import torch
+from ..utils import dump_output
+from .cases import benchmark_cases
+from .util import benchmark
+import pprint
+
+
+BM_NAME = 'functorch'
+
+
+def run_benchmarks():
+    metrics = {}
+
+    for case_ctor in benchmark_cases:
+        case = case_ctor()
+        runtime_ms = benchmark(case)
+        metrics[case.name()] = runtime_ms
+    return metrics
+
+
+def run(args):
+    metrics = run_benchmarks()
+    result = {
+        'name': BM_NAME,
+        'environ': {
+            'git_version': torch.version.git_version,
+        },
+        'metrics': metrics,
+    }
+    pprint.pprint(result)
+    dump_output(BM_NAME, result)
```

Comment:
From reading the other user benchmarks, it looks like the userbenchmark CI expects us to dump a json here. Is that correct?

Also, once the json has been dumped, does userbenchmark send the information to scribe?

Reply:
Correct, the CI runs a command like `pytorch run_benchmark.py functorch`, then expect the json file to exist in the path like `benchmark/.userbenchmark/functorch/metric-YYMMDDHHMMSS.json`, then it will analyze the json and upload the result to Amazon S3 and scribe.